tests/test_core.py

```diff
@@ -4471,8 +4471,8 @@ def test_dylink_load_compiled_side_module(self):
 
   @needs_dylink
   def test_dylink_dso_needed(self):
-    def do_run(src, expected_output):
-      self.do_run(src + 'int main() { return test_main(); }', expected_output)
+    def do_run(src, expected_output, **kwargs):
```

Comment:
Is this for `emcc_args` too? If so why is here not `emcc_args` as in other functions?

Reply:
Just because there is no need I guess? ... I don't care about that particular argument in this case and can just forward it to the inner `do_run`. 

I think are right though, its cleaner just to pass emcc_args...  I'll fix that.